lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -330,31 +330,31 @@ function dosomething_rogue_previously_failed($values) {
  * Update the timestamp and tries in the dosomething_rogue_failed_task_log for a failed reportback item.
  *
  * @param array $values
- * @param array $previously_failed
+ * @param object $previously_failed
  *
  */
 function update_failed_task_log($values, $previously_failed) {
   // Increment the number of tries the cron job attempted to send failed record to Rogue.
-  $incremented_tries = $previously_failed[0]->tries + 1;
+  $incremented_tries = $previously_failed->tries + 1;
```

Comment:
This is exactly what I meant! So on the 2nd try overall, it grabs `tries` which is `NULL`. Then it adds 1 to that (I guess you can do that if it's not crashing) to get 1?